docs/en/pact-functions.md

```diff
@@ -5,14 +5,14 @@
 
 Constant denoting the ASCII charset
 
-Constant: 
+Constant:
```

Comment:
please regen to remove whitespace changes

Reply:
:+1: